archiva-modules/plugins/stage-repository-merge/src/main/java/org/apache/archiva/stagerepository/merge/Maven2RepositoryMerger.java

```diff
@@ -194,9 +194,6 @@ private void createFolderStructure( String sourceRepoId, String targetRepoId, Ar
         copyFile( sourceArtifactFile, targetArtifactFile );
 
         // pom file copying
-        String fileName = artifactMetadata.getProject() + "-" + artifactMetadata.getVersion() + ".pom";
-
-        // pom file copying
         // TODO need to use path translator to get the pom file path
```

Comment:
The assign is not used and seems to have not side effect. The code is commented out a bit lower ... should that dead code block be removed as well?